hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/feedcomm/ExecuteOperationCommand.java

```diff
@@ -89,9 +89,15 @@ private ExecuteOperationResponse executeOperationDMR(ExecuteOperationRequest req
         Address opAddress = null;
         String actualOperationName = null;
 
+        String requestedOpName = request.getOperationName();
         Collection<DMROperation> ops = resource.getResourceType().getOperations();
+        MsgLogger.LOG.tracef("Searching for operation [%s] among %d operations available for resource [%s].",
+                requestedOpName, ops.size(), resource.getID());
```

Comment:
rather than logging each time in the loop, why not just log all the operations here in this one trace? that way, one trace log message provides you with all the information you need:

 MsgLogger.LOG.tracef("Searching for operation [%s] among operations [%s] for resource [%s].",
                requestedOpName, ops, resource.getID());

You can then eliminate the changed code below. Now you will have all the information in a single log message (with the added benefit of not having to make a method call ops.size() when trace isn't enabled)